### PR TITLE
Review and remove unused configs

### PR DESCRIPTION
**Pull request by deividaspetraitis**
This pull request removes obsolete configuration parameters to avoid maintanance overhead and confusion, for more information please see #243

Task scope was to update basically three files ( and the rest as a side effect ):

- `README.md`
- `config.json`
- `config-tested.json`

Additionally,  comments were added for some of the configuration data structures and their parameters to explain their purpose and meaning. Still, there are parameters that are missing documentation and I would be happy to update those as well with someone's help and suggestions.

### CODE REVIEW

File: domain/cache/export_test.go
Comment (deividaspetraitis) [inline domain/cache/export_test.go:1]: This change was introduced by `go fmt`

File: router/repository/memory_router_repository_test.go
Comment (deividaspetraitis) [inline router/repository/memory_router_repository_test.go:1]: Changes in this file were introduced by `go fmt`

File: domain/pricing.go [hunk 1]
```diff
@@ -102,11 +102,17 @@ type PricingConfig struct {
 	CacheExpiryMs int `mapstructure:"cache-expiry-ms"`
 
 	// The default quote chain denom.
+	// 0 stands for chain. 1 for Coingecko.
+	// Currently, only on-chain is supported.
```
Comment (p0mvn) [inline domain/pricing.go]: We just merged the coingecko implementation last week in #209 
```suggestion
```

**Review by p0mvn — APPROVED** [review]
Nice work, thanks!

**Pull request merged**